### DISCUSSION TITLE
如何修改组件的类名中的样式

### DISCUSSION THREAD

**Discussion by guzily** [tusen-ai/naive-ui#4139] [Q&A]
我需要修改n-image这个组件某个类名的样式，我要将预览图的背景设置为纯黑色，下面的写法错了吗？为什么不起效果呢？
`<style scoped lang="scss">
:deep(.n-image-preview-container) {
  background-color: rgb(0, 0, 0, 1) !important;
}
</style>`

**Comment by jahnli** [comment]


**Reply by guzily** [reply]
谢谢解答，但是依旧不起效果
